main.tf

```
@@ -234,6 +234,172 @@ module "cloudwatch_metrics" {
   tags = var.tags
 }
 
+################################################################################
+# EFS CSI DRIVER
+################################################################################
+
+locals {
+  efs_csi_driver_service_account = try(var.efs_csi_driver.service_account_name, "efs-csi-controller-sa")
+}
+
+data "aws_iam_policy_document" "aws_efs_csi_driver" {
+  statement {
```

lets add the create conditional here as well, so that way it doesn't show up in the terraform diff if users are not enabling it
```suggestion
  statement {
    count = var.enable_efs_csi_driver ? 1 : 0
```